README.md

```diff
@@ -77,7 +69,7 @@ pod 'StreamChatCore', '~> 2.0' # if needed
 pod 'StreamChatClient', '~> 2.0' # if needed
 ```
 
-- **Migrating `StreamChat` (UI components):** If you use our Chat UI components, **migration from 2.x is not recommended yet.** The UI components SDK was renamed to `StreamChatUI` and is currently only in public beta.
+- **Migrating `StreamChat` (UI components):** If you use our Chat UI components, check [this page](https://github.com/GetStream/stream-chat-swift/wiki/Migrating-from-1.x-and-2.x) for the Migration guide. The UI components SDK was renamed to `StreamChatUI`.
```

Comment:
nit: check the [Migration Guide](https://github.com/GetStream/stream-chat-swift/wiki/Migrating-from-1.x-and-2.x) ? ;-)

Reply:
you missed this